netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -273,14 +273,18 @@ protected void onStreamError(ChannelHandlerContext ctx, Throwable cause,
     // Close the stream with a status that contains the cause.
     Http2Stream stream = connection().stream(http2Ex.streamId());
     if (stream != null) {
-      clientStream(stream).transportReportStatus(Status.fromThrowable(cause), false,
-              new Metadata());
+      clientStream(stream).transportReportStatus(statusFromError(cause), false, new Metadata());
     }
 
     // Delegate to the base class to send a RST_STREAM.
     super.onStreamError(ctx, cause, http2Ex);
   }
 
+  private Status statusFromError(Throwable cause) {
+    return cause instanceof Http2Exception ? Status.INTERNAL.withCause(cause)
```

Comment:
Yes, any HTTP/2 error should be INTERNAL instead of UNKNOWN, OkHttp is doing like that, the [spreadsheet of error normalization](https://docs.google.com/spreadsheets/d/1hnzQmrM8ZC3iiKR9MrpvaFRMZLhm2OJ_Ltqeb_jxvrk/edit#gid=0) also says 
```
Protocol errors:	 GRPC_STATUS_INTERNAL
```